active/0000-element-and-fragment.md

```diff
@@ -0,0 +1,198 @@
+- Start Date: 2015-02-27
+- RFC PR:
+- Ember Issue:
+
+# Summary
+
+Introduce `Ember.Fragment` and the `<element>` and `<fragment>`
+template keywords.
+
+# Motivation
+
+This RFC addresses two key problems:
+
+ - Ember has no clear concept of a "tagless component". We need such a
+   concept now that we're moving to a component-centric world.
```

Comment:
@stefanpenner - We need this because we use this concept internally. There are cases where you might want to implement things like our internal `{{each}}` and `{{if}}` helpers as components, and you wouldn't need/want there to have to be an extra element.